11ty/definitions/minoritised.md

```diff
@@ -12,7 +12,7 @@ reading:
     href: https://www.theodysseyonline.com/minority-vs-minoritize
 ---
 
-groups resulting from social constructs have been granted less power or representation compared to other members or groups in society
+groups, resulting from social constructs, that have been granted less power or representation compared to other members or groups in society
```

Comment:
I'm not sure that "resulting from social constructs" is an apositive as I think the socially constructed part is critical.

I wonder if to a void the awkwardness of the original phrasing, we could instead say, "socially constructed groups that have been..." What do you think about that alternate approach?